source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -128,6 +139,17 @@ content: |
             const base64DataKeyId = key.toString('base64');
             console.log("DataKeyId [base64]: ", base64DataKeyId);
            });
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
+           connectionString = "mongodb://localhost:27017";
+           keyVaultNamespace = "encryption.__keyVault";
+           fle_opts = AutoEncryptionOpts( kms_providers , keyVaultNamespace, mongocryptd_bypass_spawn = True )
```

Comment:
I think there is an extra space after `kms_providers`. 
Also, per a prior comment:

> Would it be better to stay consistent with spacing for function calls/definitions? Here, there are spaces around the first and last parameters, but in other function calls/definitions, there are no spaces.

Reply:
Fixed:

- Removed extra space

Sorry about this spacing issue. Not sure why it keeps coming up, maybe something wrong with my vscode settings